content/docs/contributing/docs.md

```diff
@@ -57,7 +57,7 @@ the website engine itself. Source code and content files need to be properly
 formatted and linted as well, which is also ensured by the full setup below.
 
 Make sure you have [Python](https://www.python.org/downloads/) 3.8+, a recent
-LTS version of [Node.js](https://nodejs.org/en/) (`>=14.0.0`, `<=18.x`), and
+LTS version of [Node.js](https://nodejs.org/en/) (`>=18.0.0`, `<=19.x`), and
```

Comment:
Updated this due to a error I got after pulling `main` and running `yarn`:

```
error @dvcorg/gatsby-theme-iterative@0.3.0: The engine "node" is incompatible with this module.
Expected version ">=18.0.0". Got "16.17.1"
```

Not sure about the max though (19)